internal/outofband/action_handlers.go

```diff
@@ -224,9 +223,9 @@ func (h *actionHandler) downloadFirmware(a sw.StateSwitch, c sw.TransitionArgs)
 
 	// validate checksum
 	//
-	// This assumes the checksum is of type SHA256
+	// This assumes the checksum is of type MD5
 	// it would be ideal if the firmware object indicated the type of checksum.
-	if err := checksumValidateSHA256(file, action.Firmware.Checksum); err != nil {
+	if err := checksumValidateMD5(file, action.Firmware.Checksum); err != nil {
```

Comment:
Why change from SHA256 to MD5? The latter is trivially spoofed, and I don't think hash-speed is going to be a rate-limiting factor here.

Reply:
Agreed in point - for now this is the data Serverservice provides, we need to update the `firmware-syncer` component to use SHA256 instead.